resources/js/Pages/Item/Detail.tsx

```diff
@@ -0,0 +1,284 @@
+    purChase: '',
+    selling: '',
+    labels: [
+      { label: 'ラベル1', labelValue: '値2' },
+      { label: 'ラベル2', labelValue: '値2' },
+      { label: 'ラベル3', labelValue: '値3' },
+      { label: 'ラベル4', labelValue: '値4' },
+      { label: 'ラベル5', labelValue: '値5' },
+      { label: 'ラベル6', labelValue: '値6' },
+      { label: 'ラベル7', labelValue: '値7' },
+      { label: 'ラベル8', labelValue: '値8' },
+    ],
+    purChasePrice: 0,
+    sellingPrice: 0,
+  })
+
+  // 追加機能は別ブランチ対応だが、書いてしまったのでとりあえず残しておく
+  // const addLabels = (event:any) => {
+  //     setStates({...states, labels:[
+  //         ...states.labels,
+  //         {
+  //             label: event.target.value,
+  //             labelValue: event.target.value
+  //         }
+  //     ]})
+  // }
+
+  const handleStatusChange = (event: SelectChangeEvent) => {
+    setStates({ ...states, status: event.target.value })
+  }
+
+  const handleBrandChange = (event: SelectChangeEvent) => {
+    setStates({ ...states, brand: event.target.value })
+  }
+
+  const handleCategoryChange = (event: SelectChangeEvent) => {
+    setStates({ ...states, category: event.target.value })
+  }
+
+  const handleColorChange = (event: SelectChangeEvent) => {
+    setStates({ ...states, color: event.target.value })
+  }
+
+  const handleTemplateChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setStates({ ...states, template: event.target.value })
+  }
+
+  const handlePurChaseChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setStates({ ...states, purChase: event.target.value })
+  }
+
+  const handleSellingChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setStates({ ...states, selling: event.target.value })
+  }
+
+  const handlePurChasePriceChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    if (isNaN(Number(event.target.value))) return
+    if (!Number.isSafeInteger(Number(event.target.value))) return
+    setStates({ ...states, purChasePrice: Number(event.target.value) })
+  }
+
+  const handleSellingPriceChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    if (isNaN(Number(event.target.value))) return
+    if (!Number.isSafeInteger(Number(event.target.value))) return
+    setStates({ ...states, sellingPrice: Number(event.target.value) })
+  }
+
+  return (
+    <>
+      <Button href="/item" startIcon={<KeyboardArrowLeftIcon />}>
+        商品一覧
+      </Button>
+      <Grid container mt={2}>
+        <Grid item xs={12} lg={6}>
+          <Grid item xs={12} lg={11}>
+            <Card>
+              <CardMedia
+                component="img"
+                image="https://placehold.jp/150x150.png"
+                alt="dammy-img"
+              />
+            </Card>
+            <Box component="div" mt={2}>
+              <SelectBox
+                title={statusTitle}
+                selectNames={statusNames}
+                state={states.status}
+                handleChange={handleStatusChange}
```

Comment:
```suggestion
                onChange={HandleStatusChange}
```
@masato5579 ここは要議論ですが...イベントハンドラの命名ルールは色々ありますが、[ドキュメント](https://ja.reactjs.org/docs/handling-events.html)によると上記が推奨されてるみたいですね。  

ちなみに、ゴルフ案件(vue3)では

```
onChanged={onStatusChanged}
```
みたいな書き方してました。

どれがいいですかね？

Reply:
@YutaManaka 

onChangeが無難だったので、直します！